hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/PlanExecutor.java

```diff
@@ -504,7 +506,7 @@ SqlResult execute(ExplainStatementPlan plan) {
         );
     }
 
-    SqlResult execute(SelectPlan plan, QueryId queryId, List<Object> arguments, long timeout) {
+    SqlResult execute(SelectPlan plan, QueryId queryId, List<Object> arguments, long timeout, @Nullable SqlSecurityContext ssc) {
```

Comment:
`SecurityContext` and `SqlSecurityContext` should be `@Nonnull`. In case of OS and member invocation we use `NoOpSqlSecurityContext.INSTANCE`. That will simplify code even further